src/elements/content-preview/ContentPreview.js

```diff
@@ -742,6 +743,11 @@ class ContentPreview extends React.PureComponent<Props, State> {
             fileOpts[fileId].fileVersionId = selectedVersion.id;
         }
 
+        if (activeAnnotationId) {
```

Comment:
Did we decide on if/how to standardize between active/current/selected?

Reply:
No, I'll try to address it in a follow up PR